chroot-bin/volume

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+
+# This script uses cras_test_client to present an interface similar to the
+# brightness script for crouton.
+
+# The MIT License (MIT)
```

Comment:
This needs to be licensed like the rest of the project, and you'll need to sign the [CLA](https://cla.developers.google.com/clas/new?kind=KIND_INDIVIDUAL&domain=DOMAIN_GOOGLE) for the code to be accepted.